src/platform/plugins/shared/chart_expressions/expression_xy/public/components/xy_chart.test.tsx

```diff
@@ -964,7 +964,8 @@ describe('XYChart component', () => {
     expect(component.find(Settings).prop('rotation')).toEqual(90);
   });
 
-  test('it renders regular bar empty placeholder for no results', () => {
+  // TODO: `component.find(EmptyPlaceholder)` is not working as expected with emotion css prop babel present
+  test.skip('it renders regular bar empty placeholder for no results', () => {
```

Comment:
Seems like shallow rendering is broken here and it is no longer possible to reach `EmptyPlaceholder` because it wasn't rendered in shallow mode

Reply:
it was rendered it was just transformed into a different representation of an element, you can still reach it via data-test-subj locator

so same as https://github.com/elastic/kibana/pull/216489/files#r2028896170

Reply:
a better approach in this file was to just use `mountWithIntl` which passes the tests

Reply:
done